bitshares-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
+      "type": "string",
+      "defaultValue": "From_PPA",
+      "allowedValues": [
+        "From_PPA",
+        "From_Source"
+      ],
+      "metadata": {
+        "description": "Method to install bits. From_PPA: unofficial community bianaries.  From_Source: official BitShares repo on GitHub."
+      }
+    },    
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A2",
+      "allowedValues": [
+        "Standard_A1",
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_D1_V2",
+        "Standard_D2_V2",
+        "Standard_D3_V2"
+      ],
+      "metadata": {
+        "description": "Size of VM"
+      }
+    }
+  },
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "OSDiskName": "BitSharesOSDisk",
+    "imageSKU": "15.10",
```

Comment:
This template will not currently build on 14 LTS due to Boost version incompatibility (need 1.58+). I plan to continue to maintain the template and use then current builds (15+).

Will this be acceptable?

Cheers,
Fox

On Mar 5, 2016, at 8:50 AM, Kay Singh <notifications@github.com<mailto:notifications@github.com>> wrote:


In bitshares-ubuntu-vm/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/1335#discussion_r55094039>:

> +        "Standard_A2",
> +        "Standard_A3",
> +        "Standard_D1_V2",
> +        "Standard_D2_V2",
> +        "Standard_D3_V2"
> +      ],
> +      "metadata": {
> +        "description": "Size of VM"
> +      }
> +    }
> +  },
> +  "variables": {
> +    "imagePublisher": "Canonical",
> +    "imageOffer": "UbuntuServer",
> +    "OSDiskName": "BitSharesOSDisk",
> +    "imageSKU": "15.10",


Can this run on 14.04 LTS? 15.10 will EOL'd in a few months and then the image will be removed from Azure causing this template to break

-
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/1335/files#r55094039>.